ui/components/ui/settings-icons/advanced-icon/advanced-icon.component.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+export default function GeneralIcon() {
```

Comment:
Is there some reason we made all of these icons into react components rather than just passing their path's as the `src` in `img` tags?